src/app/containers/ATIAnalytics/params/media/index.jsx

```diff
@@ -0,0 +1,27 @@
+import { useContext } from 'react';
+import path from 'ramda/src/path';
+
+import atiPageViewParams from '../../atiUrl';
+import { RequestContext } from '../../../../contexts/RequestContext';
+import { ServiceContext } from '../../../../contexts/ServiceContext';
+
+const Params = pageData => {
+  const { platform, statsDestination } = useContext(RequestContext);
+  const { atiAnalyticsAppName, service } = useContext(ServiceContext);
+
+  const { metadata } = pageData;
+
+  return atiPageViewParams({
+    appName: atiAnalyticsAppName,
+    contentId: metadata.id,
+    contentType: 'player-live',
```

Comment:
This should likely come from the `metadata` rather than being hardcoded, but this can be changed in the future